test/parser/parser_test.cpp

```diff
@@ -404,6 +448,19 @@ TEST_F(ParserTestBase, OperatorTest) {
     EXPECT_EQ(expr->GetReturnValueType(), type::TypeId::BOOLEAN);
   }
 
+  {
+    std::vector<std::string> queries;
+
+    // Coverage for NullNodeTransform
+    queries.emplace_back("SELECT * FROM foo WHERE 0 IS NULL;");
+    queries.emplace_back("SELECT * FROM foo WHERE 0*1 IS NULL;");
+    queries.emplace_back("SELECT * FROM foo WHERE ? IS NULL;");
+
+    for (const auto &query : queries) {
+      auto stmt_list = pgparser.BuildParseTree(query);
```

Comment:
We need to be checking something here. We're building the parse tree so this is a false code coverage result